### PR TITLE
User/v jandrew/clarify xbox support 

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**

### CODE REVIEW

File: microsoft-edge/webview2/concepts/user-data-folder.md [hunk 1]
```diff
@@ -416,7 +416,7 @@ On this platform, use the default UDF location.
 
 **Example code:**
 
-For example code, see the WinUI 3 `.cs` file, at [WebView2Samples repo > WebView2_WinUI3_Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2_WinUI3_Sample).
+For example code, see the WinUI 3 `.cs` file, at [WebView2Samples repo > WebView2_WinUI3_Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI3_GettingStarted).
```
Comment (captainbrosset) [inline microsoft-edge/webview2/concepts/user-data-folder.md:419]: Weren't the changes in this file already done in a previous PR that we merged? You might need to update your main branch and merge it in this PR branch to make sure this PR branch is up to date.

File: microsoft-edge/webview2/roadmap.md [hunk 2]
```diff
@@ -56,3 +56,6 @@ You can access WebView2 controls in your UWP applications using [Windows UI Libr
 ### Windows UI Library 3
 
 You can access WebView2 controls in your applications using [Windows UI Library 3 (WinUI 3)](/uwp/toolkits/winui3/index) with the Windows App SDK.  This has reached GA.
+
+### Xbox, HoloLens, and XAML limitations
```
Comment (captainbrosset) [inline microsoft-edge/webview2/roadmap.md:60]: Please add an empty line between this h3 header and the paragraph below it.

File: microsoft-edge/webview2/get-started/winui2.md [hunk 3]
```diff
@@ -302,6 +302,9 @@ WebView2 sends URLs that are navigated to in your application to the [SmartScree
 
 This environment variable must be set prior to `CoreWebView2` creation, which occurs when the [WebView2.Source property](/windows/winui/api/microsoft.ui.xaml.controls.webview2.source) is initially set or the [WebView2.EnsureCoreWebView2Async method](/windows/winui/api/microsoft.ui.xaml.controls.webview2.ensurecorewebview2async) is initially called.
 
+#### Xbox, HoloLens, and XAML limitations
+This version of WebView 2 will only work on PC-class devices, offering the full range of capability found in the WinUI 3 variant. Xbox, HoloLens, and XAML Island support require quite a bit of additional work. These devices/scenarios may be considered at a future point in time.
```
Comment (captainbrosset) [inline microsoft-edge/webview2/get-started/winui2.md:306]:
```suggestion
This version of WebView 2 will only work on PC-class devices, offering the full range of capability found in the WinUI 3 variant. Xbox, HoloLens, and XAML Island support requires additional work, and these devices and scenarios may be considered at a future point in time.
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/concepts/user-data-folder.md (hunk 1 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/webview2/concepts/user-data-folder.md:419]: My error.  Not sure why it was added to this pr, but I removed it.

Comment (JasonAndrewWriter) [reply microsoft-edge/webview2/concepts/user-data-folder.md:419]: I removed this file.  Must have been added accidentally.

File: microsoft-edge/webview2/get-started/winui2.md [hunk 4]
```diff
@@ -302,6 +302,9 @@ WebView2 sends URLs that are navigated to in your application to the [SmartScree
 
 This environment variable must be set prior to `CoreWebView2` creation, which occurs when the [WebView2.Source property](/windows/winui/api/microsoft.ui.xaml.controls.webview2.source) is initially set or the [WebView2.EnsureCoreWebView2Async method](/windows/winui/api/microsoft.ui.xaml.controls.webview2.ensurecorewebview2async) is initially called.
 
+#### Xbox, HoloLens, and XAML limitations
```
[the remark this answers is not in the document]
Comment (JasonAndrewWriter) [reply microsoft-edge/webview2/get-started/winui2.md:305]: fixed

File: microsoft-edge/webview2/roadmap.md (hunk 2 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/webview2/roadmap.md:60]: fixed

**Review by JasonAndrewWriter — COMMENTED** [review]
I made your fixes

**Comment by JasonAndrewWriter** [comment]
Closing this one as this is the current pr.  https://github.com/MicrosoftDocs/edge-developer/

**Pull request closed without merging**